code/game/machinery/camera/camera.dm

```diff
@@ -13,7 +13,7 @@
 	var/c_tag_order = 999
 	var/status = 1
 	anchored = 1.0
-	var/invuln = null
+	var/invuln = 0

```

Comment:
This is just me nitpicking so mappers know they can set a value to 1 for it to be invulnerable.